torch/csrc/jit/graph_executor.h

```diff
@@ -20,7 +20,7 @@ struct ExecutionPlanState {
 };
 
 struct GraphExecutorState {
-  const Graph* graph;
+  const Graph* graph = nullptr;
```

Comment:
This seems to imply that this field is nullable, while in reality you should *never* find a `nullptr` there.

Reply:
null > garbage

Reply:
The point is you should never see the `null`. If you are really that afraid about garbage, you can add a constructor to enforce that this field is always correctly initialized.

Reply:
But this is the same thing as adding such a constructor, just that it always applies (and doesn't apply if you construct it with another value in the constructor)? I really don't understand why you're so against this check. If I add another constructor and forget to initialize it to a non-garbage pointer and my program crashes sometime later because the pointer points to garbage, that's what you prefer over having it always be null? I really don't understand that at all. We're talking about "always safe, maybe sometimes unnecessarily so" vs. "sometimes unsafe". I don't understand how the former is not strictly preferred in a large codebase